src/zenml/client.py

```diff
@@ -3896,7 +3896,7 @@ def list_pipeline_runs(
         tag: Optional[str] = None,
         tags: Optional[List[str]] = None,
         user: Optional[Union[UUID, str]] = None,
-        run_metadata: Optional[Dict[str, Any]] = None,
+        run_metadata: Optional[List[str]] = None,
```

Comment:
Does this change also require some updates to the docs?

Reply:
We didn't have any docs specifically pointing out how to filter by run_metadata. Since we are will be merging the pages really soon anyhow, I will handle that in an upcoming PR.